packages/application-shell-connectors/package.json

```diff
@@ -39,10 +39,10 @@
     "lodash-es": "4.17.11",
     "moment-timezone": "^0.5.23",
     "prop-types": "15.7.2",
-    "recompose": "0.26.0"
+    "recompose": "^0.30.0"
```

Comment:
it's mostly performance optimizations, no breaking changes 🤔

Reply:
This is a bit dangerous. In the mc-fe this version breaks the Modified Products view. I never understood why. But I would recommend playing it safe and keep it as is. Maybe update in the mcfe first to ensure the version works. Whatever causs things to break...

Reply:
thanks. Should we add a `throw-if-pinned-deps-changed` to app kit as well then?

Reply:
Sure. I always wondered why it is not in yarn natively or why there isnt a pkg for this. Could just be an entry in the pkg.json called noUpdate or something 🙃

Reply:
FYI: there is an issue to work on that (#100)

As for `recompose`, I think we can completely remove it as we only use it for the display name.

I'll push a commit

Reply:
Actually I'll open a new PR, as there are a few more changes.